internal/schema/private_endpoint_command.go

```diff
@@ -0,0 +1,40 @@
+package schema
+
+import "github.com/hashicorp/terraform-plugin-framework/types"
+
+type AWSCommandRequest struct {
+	// ClusterId is the ID of the cluster for which the collection needs to be created.
```

Comment:
Update docstrings in this file, it mentions collection everywhere

Reply:
addressed in 9a0365f